packages/console-types/src/types/catalog/custom-resource-definitions/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2025 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CatalogCRDManifest } from '../well-known-items'
+
+export type PublicCatalogCRD = CatalogCRDManifest & {
+  isLatest: true
+  tenantId: 'mia-platform',
+  visibility: { public: true },
+}
```

Comment:
Those information, specially the tenantId, should be defined by the user I suppose.
e.g. the tenantId could be changed during the installation of Console

Reply:
definitely, this sounds like something that must come from the outside and not be wired in the lib

Reply:
This is just an internal type for our CRDs, it is not exposed in the index nor is intended to be used. I thought of it as a way of checking the CRDs written in the library. We can remove it without issues though

Nonetheless, I think our CRDs should come with these values set (as they are right now in the Console Version Upgrader)

Reply:
>This is just an internal type for our CRDs, it is not exposed in the index nor is intended to be used. I thought of it as a way of checking the CRDs written in the library. We can remove it without issues though

OK, I'd consider stating clearly that is file is for test purposes and not meant to be exposed.

>Nonetheless, I think our CRDs should come with these values set (as they are right now in the Console Version Upgrader)

this is true but shall be controlled by the installation script.. Being in this library already makes the user aware that these are "Mia-Platform CRDs" :)

Reply:
We agreed to keep this definition to make the CRDs syntactically correct, let's remove the `isLatest` and make the installation script deal with adapting the data